client/js/components/procedure/admin/NewBlueprintForm.vue

```diff
@@ -120,7 +120,7 @@
 
         <p
           v-if="isCustomerMasterBlueprintExisting && hasPermission('feature_admin_customer_master_procedure_template')"
-          class="lbl__hint u-ml-0_75 u-mb">
+          class="flash-warning lbl__hint p-1 ml-4 mb-4 mt-1">
```

Comment:
We could also use DpInlineNotification here:

```
      <dp-inline-notification
          v-if="isCustomerMasterBlueprintExisting && hasPermission('feature_admin_customer_master_procedure_template')"
          :message="Translator.trans('explanation.customer.masterblueprint.uncheck.existing')"
          type="warning" />
```

Reply:
good suggestion
8bbf938d21ae91c40efe848c87f3e6285e658043